application/src/main/java/org/togetherjava/tjbot/routines/ModAuditLogRoutine.java

```diff
@@ -33,6 +33,8 @@
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
+import static org.togetherjava.tjbot.moderation.ModAuditLogWriter.getModAuditLogChannel;
```

Comment:
static method imports should be avoided, they cause confusion. Instead, impor tthe class and then call it normally like `ModAuditLogWriter.getModAuditLogChannel()...`

Reply:
fixed that.